interface/src/app/gene/search/directives.js

```diff
@@ -0,0 +1,321 @@
+angular.module('adage.gene.search.directives', [
```

Comment:
Ah yes, you're totally right, I had totally forgotten about that syntax. As we talked about in person this morning though, it looks like grunt-uglify is minifying and loading files in alphabetical order, so that can cause a problem if the file that defines the module is not the first alphabetically. I will look more into solutions to that this morning, and if I don't find anything, we can go with the differently named modules.